index.html

```diff
@@ -10,234 +10,252 @@
+            <li>
+              <img src="image/css3.svg" alt="" />
+              CSS
+            </li>
+          </ul>
+        </li>
+        <li class="topic item">Frameworks</li>
+        <li class="topic item">Skills</li>
+      </ul>
     </div>
-    <div class="menuSection">
-      <a href="#" target="_blank">LET'S CONNECT</a>
-      <div class="socialMedia">
-        <img src="image/Twitter.png" alt="" />
-        <img src="image/Linkedin.png" alt="" />
-        <img src="image/Meduim.png" alt="" />
-        <img src="image/Github.png" alt="" />
-        <img src="image/angellist.png" alt="" />
+
+    <footer class="footer">
+      <div class="contact-me">
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I’d love
+          to help with it
+        </p>
       </div>
-    </div>
-    <div class="project">Get my resume</div>
-
-    <ul>
-      <li>
-        <div class="topic item">Languages</div>
-        <ul class="languages">
-          <li> <img src="image/js.svg" alt=""> JavaScript</li>
-          <li> <img src="image/html5.svg" alt=""> HTML</li>
-          <li> <img src="image/css3.svg" alt=""> CSS</li>
-        </ul>
-      </li>
-      <li class="topic item">Frameworks</li>
-      <li class="topic item">Skills</li>
-    </ul>
-  </div>
 
-  <footer class="footer">
-    <div class="contact-me">
-      <h2>Contact me</h2>
-      <p>
-        If you have an application you are interested in developing, a feature that you need built or a project that
-        needs coding. I’d love to help with it
-      </p>
-    </div>
+      <div class="contact-forms">
+        <form class="message-form" action="https://formspree.io/f/xqknrzed" method="post">
 
-    <div class="contact-forms">
-      <form class="message-form" action="https://formspree.io/f/xqknrzed" method="post">
+          <div class="input-li">
+            <input type="text" name="text" id="text-input" placeholder="Yeremias" maxlength="30" required>
+          </div>
+          <div class="input-li">
+            <input type="email" name="email" id="mail" placeholder="natajay14@gmail.com" required>
+          </div>
+          <div class="text-area-li">
+            <textarea name="msg" id="msg" maxlength="500" placeholder="Write your messages here" required></textarea>
 
-        <div class="input-li">
-          <input type="text" name="text" id="text-input" placeholder="Yeremias" maxlength="30" required>
-        </div>
-        <div class="input-li">
-          <input type="email" name="email" id="mail" placeholder="natajay14@gmail.com" required>
-        </div>
-        <div class="text-area-li">
-          <textarea name="msg" id="msg" maxlength="500" placeholder="Write your messages here" required></textarea>
+          </div>
+          <div class="submit-li">
+            <button type="submit">Get it Touch</button>
```

Comment:
- [ ] Kindly according to the requirements add here all the button properties for the hover, pressed, and disabled as shown in Figma and because it is very good for user experience (UX)✅✅💯💯


- [ ] Kindly make the font color for the button as shown in the Figma since I see it black in your project 💯✅